src/openforms/payments/contrib/ogone/models.py

```diff
@@ -63,10 +61,3 @@ def clean(self):
 
     def __str__(self):
         return self.label
-
-    def save(self, *args, **kwargs):
-        super().save(*args, **kwargs)
-
-        from openforms.config.models import CSPSetting
-
-        CSPSetting.objects.set_for(self, [(CSPDirective.FORM_ACTION, self.endpoint)])
```

Comment:
is the `CSPSetting.objects.set_for` method still used anywhere else after this removal? Perhaps we can remove that?

Reply:
It is used in the `update_analytics_tool` function